packages/server/lib/controllers/flow.controller.ts

```diff
@@ -58,7 +58,12 @@ class FlowController {
                 return;
             }
 
-            await syncManager.triggerIfConnectionsExist(preBuiltResponse.result, environment.id, logContextGetter, orchestrator);
+            await syncManager.triggerIfConnectionsExist({
+                flows: preBuiltResponse.result,
+                environmentId: environment.id,
+                logContextGetter,
+                orchestrator
+            });
```

Comment:
just changing the parameters into an object for now. At some point,  I will have to pass the variant